CompoundV3-Bots/Governance-Events/README.md

```diff
@@ -0,0 +1,77 @@
+# Governance Events Bot
+
+## Description
+
+This bot detects events related to governance actions in Compound v3 Comet contracts.
+
+The Comet deployment addresses for each network can be configured in the
+`agent.config.ts` file.
+
+## Supported Chains
+
+- Ethereum
+- Polygon
+
+## Alerts
+
+- COMP2-2-1
+  - Fired when there's a pause action on a Comet contract
+  - Severity is always set to "Medium"
+  - Type is always set to "Info"
+  - Metadata:
+    - `chain`: Network chain ID or name
+    - `comet`: Address of the related Comet contract
+    - `supplyPaused`: Status of supply pausing
+    - `transferPaused`: Status of supply pausing
+    - `withdrawPaused`: Status of supply pausing
+    - `absorbPaused`: Status of supply pausing
+    - `buyPaused`: Status of supply pausing
```

Comment:
All descriptions are set to `supply pausing`, please update them to match the action

Reply:
Good catch, fixed on [165698a](https://github.com/NethermindEth/Forta-Agents/pull/371/commits/165698a8b3e69086bf503179dc871bb2fd347501).